### DISCUSSION TITLE
Command bus upgrade to 9

### DISCUSSION THREAD

**Discussion by ossigeno** [mako-framework/framework#305] [General]
Previously the handle signature was
public function handle(CommandInterface $command)

now is
public function handle(CommandInterface $command) : mixed

What is the handle expected to return? There is nothing mentioned here:
https://makoframework.com/docs/9.0/learn-more:command-bus

**Comment by freost** [comment]
It does say that it can have a return value:
```
The CommandHandlerInterface::handle() method can return data that can be used to handle successes and errors.
```
But I agree that enforcing `mixed` when you're not returning anything looks bad so I'll remove the return type from the interfaces.
